ts/features/wallet/onboarding/common/searchBank/SearchStartComponent.tsx

```diff
@@ -85,17 +85,16 @@ export const SearchStartComponent: React.FunctionComponent<Props> = (
       </Body>
 
       <View spacer={true} large={true} />
-      <Body>
+      <Body
```

Comment:
Could this code change the element behavior just [like done here](https://github.com/pagopa/io-app/pull/3246#discussion_r674560400)? 
I guess this code makes tappable the whole block. Only the link should be tappable

Reply:
@andrea-favaro any chances we can act on this today? The size and scope of this PR would allow it to be merged soon.

Reply:
@pietro909 Sure, I left a comment to @Undermaken here https://github.com/pagopa/io-app/pull/3246, it's the same situation as this PR

Reply:
@thisisjp what's your opinion about these changes?
TL;DR this PR fixes the accessibility issue and makes the whole element tappable (not only the link)
<img width="402" alt="Schermata 2021-07-23 alle 09 25 05" src="https://user-images.githubusercontent.com/822471/126750003-d6e25b83-98df-4da4-b61c-29e488bd2fde.png">
Is it ok for you?

Reply:
The goal here is:
1. being able to tap on a paragraph and read the full content
2. Being able to understand there’s a link, in case I’m interested in
learning more
3. Being able to open that link

If I can achieve this goal, then let’s proceed :)